resources/assets/js/features/trip/components/Forms/TripForm.js

```diff
@@ -6,8 +6,9 @@ import { localize } from 'react-localize-redux';
 
 import Waypoints from './Waypoints';
 import Input from 'app/components/Input';
+import {InputDateTime} from 'app/components/Controls/index.js';
```

Comment:
no need to specify index.js, you can use app/components/Controls path to export needed prop